fawltydeps/settings.py

```diff
@@ -114,7 +135,7 @@ class Settings(BaseSettings):  # type: ignore
     pyenvs: Set[Path] = {Path(".")}
     custom_mapping: Optional[CustomMapping] = None
     ignore_undeclared: Set[str] = set()
-    ignore_unused: Set[str] = set()
+    ignore_unused: Set[str] = default_ignore_unused
```

Comment:
To reflect the above:

```suggestion
    ignore_unused: Set[str] = DEFAULT_IGNORE_UNUSED
```

Reply:
Addressed in https://github.com/tweag/FawltyDeps/pull/365/commits/b6afa98689e4720130fad434d0ce13e2d2b44a26.